blob.py

```diff
@@ -119,9 +120,19 @@ def download(context, backend, handle, output, format):
     type=click.Choice(["GENERIC", "RANDOM"], case_sensitive=False),
 )
 @click.option("--tempo", default=1.0, type=float)
+@click.option(
+    "--tracks",
+    default="AUTO",
+    type=str,
+)
+@click.option(
+    "--phoneme_fill_in",
+    default="SIL",
+    type=click.Choice(["SIL", "A", "E", "I", "O", "U"], case_sensitive=False)
+)
 @click.pass_obj
 @click.pass_context
-def create(context, backend, input, output, format, theme, tempo, language):
+def create(context, backend, input, output, format, theme, tempo, language, tracks, phoneme_fill_in):
```

Comment:
I plan to use [Black](https://github.com/psf/black) for formatting, so let's anticipate to their long line splitting:

```suggestion
def create(
    context,
    backend,
    input,
    output,
    format,
    theme,
    tempo,
    language,
    tracks,
    fill
):
```